.pa11yci.conf.js

```diff
@@ -46,7 +46,8 @@ var config = {
     },
     // see https://github.com/grafana/grafana/pull/41693#issuecomment-979921463 for context
     // on why we're ignoring singleValue/react-select-*-placeholder elements
-    hideElements: '#updateVersion, [class*="-singleValue"], [id^="react-select-"][id$="-placeholder"]',
+    hideElements:
+      '#updateVersion, [class*="-singleValue"], [id^="react-select-"][id$="-placeholder"], div[aria-role=presentation]',
```

Comment:
same here

Reply:
fixed

Reply:
in their example docs they don't quotes it 🤷